dev-build/conf/routes

```diff
@@ -92,6 +92,7 @@ GET            /most-read-facebook.json
 GET            /most-read-twitter.json                                                                                           controllers.MostViewedSocialController.renderMostViewed(socialContext: String = "twitter")
 GET            /most-read.json                                                                                                   controllers.MostPopularController.render(path = "")
 GET            /most-read/*path.json                                                                                             controllers.MostPopularController.render(path)
+GET            /most-read-amp.json                                                                                               controllers.MostPopularController.renderPopularAmp()
```

Comment:
Thinking that these endpoints (there'll be more of them) should be named by format, rather than by client. For example: `/most-read-schema.json`